crates/xtask/src/codegen.rs

```diff
@@ -132,7 +134,9 @@ impl CodegenCommand {
     fn generate_function_bindings<P: AsRef<Path>>(header_file: P) -> Result<bindgen::Bindings> {
         bindgen::Builder::default()
             .header(header_file.as_ref().to_string_lossy())
-            .allowlist_function("ie_.*")
+            .clang_arg("-I/home/rahul/repos/openvino-binding/openvino-rs/crates/openvino-sys/upstream/src/bindings/c/include")
```

Comment:
Ditto

Reply:
Done, changed to `.clang_arg("-I./crates/openvino-sys/upstream/src/bindings/c/include")`